google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/stresstest/DatabaseClientStressTest.java

```diff
@@ -0,0 +1,548 @@
+/*
```

Comment:
This is great, but I don't think this is the appropriate place to put this type of test alongside other JUnit tests. For example, we have a different module for our conformance tests - not sure if we should think about a new module for stress testing.

That said, can we move this to a future PR after we think about it some more?